Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM kbase/runtime:latest
+
+WORKDIR /opt/run/
+
+# set the GOPATH
+ENV GOPATH=/opt/go PATH=$PATH:$GOPATH/bin
+RUN mkdir /opt/go
+
+# install dependencies for SHOCK
+RUN \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
+    echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list && \
+    apt-get update && \
+    apt-get install -y \
+        libsasl2-dev libsasl2-modules-gssapi-mit \
+        mongodb-10gen=2.4.14 \
+        supervisor \
+        expect
+
+# update golang to 1.3.x
```

Comment:
Really? Shock is only at Go 1.3? Current release is 1.7! Or is KBase just running on a version of Shock that only runs on 1.3?